pkg/github/github.go

```diff
@@ -6,20 +6,25 @@ import (
 	"regexp"
 	"sort"
 	"strconv"
+	"time"
 
 	"github.com/giantswarm/microerror"
 
+	"github.com/giantswarm/config-controller/pkg/github/internal/cache"
 	"github.com/giantswarm/config-controller/pkg/github/internal/gitrepo"
 	"github.com/giantswarm/config-controller/pkg/github/internal/graphql"
 )
 
 type Config struct {
-	Token string
+	Token           string
+	CacheExpiration time.Duration
 }
 
 type GitHub struct {
 	graphQLClient *graphql.Client
 	repo          *gitrepo.Repo
+	tagCache      *cache.TagCache
+	storeCache    *cache.StoreCache
```

Comment:
I hoped we can cache Store instances outside of this package. I don't think it makes much sense to cache in CLI mode.

Following this pattern would make sense to me https://github.com/giantswarm/aws-operator/blob/master/service/internal/images/images.go so we cache only for the handler needs.

Reply:
Good call.